docs/static/resources/openapi.json

```
@@ -2075,7 +2075,7 @@
             "type": "string"
           },
           "query_context_generation": {
-            "description": "The query context generation represents whether the query_contextis user generated or not so that it does not update user modfiedstate.",
+            "description": "The query context generation represents whether the query_contexts user generated or not so that it does not update user modifiedstate.",
```

```suggestion
            "description": "The query context generation represents whether the query_context is user generated or not so that it does not update user modified state.",
```
I think this is what the author was going for?